### ISSUE TITLE
Unable to run controller on RKE

### ISSUE THREAD

**Issue by xrl** [cert-manager/trust-manager#69]
We are on RKE with Kubernetes 1.21. When I install the helm chart I see the replicaset is stuck with 0 copies and this attached event:

```
% % k -n cert-manager describe rs cert-manager-trust-6796dcc8f5
Events:
  Type     Reason        Age                      From                   Message
  ----     ------        ----                     ----                   -------
  Warning  FailedCreate  4m32s (x548 over 5d20h)  replicaset-controller  Error creating: pods "cert-manager-trust-6796dcc8f5-" is forbidden: PodSecurityPolicy: unable to admit pod: [pod.metadata.annotations[container.seccomp.security.alpha.kubernetes.io/cert-manager-trust]: Forbidden: seccomp may not be set pod.metadata.annotations[container.seccomp.security.alpha.kubernetes.io/cert-manager-trust]: Forbidden: seccomp may not be set]
```

and it does not appear that seccomp is a configurable field on a PSP. So I'm not sure how to allow the pod to define seccomp. Would the project be open to a flag for optionally hiding the securityContext option?

**Comment by SgtCoDFish** [comment]
Hey, sorry this took a while to get back to you on!

Thanks for raising this and submitting a PR, you're a star! Seems reasonable to add this, I'll take a look at #70 :+1:

**Comment by xrl** [comment]
How do we feel about cutting a release?

**Comment by SgtCoDFish** [comment]
I think we'll cut a release after #46 gets merged because that's a pretty big new usability improvement. If it looks like that might not be merged any time soon, I'll cut 0.3.1 to release this.

Definitely not until at least next week though!

**Comment by xrl** [comment]
Perfect, we'll keep using our local copy. No problem.
